### PR TITLE
Add d.ts. file, closes #2

### PR DESCRIPTION
**Pull request by 1999**

### CODE REVIEW

**Comment by 1999** [comment]
@torifat could you pls also take a look at this PR?

File: src/typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  export interface IOutputError {
+    error: string;
+    line: number;
+    column: number;
+    suggestion?: string;
+  }
+
+	export interface IBetterAjvErrors {
```
Comment (torifat) [inline src/typings.d.ts]: Is this expected?

**Review by torifat — COMMENTED** [review]
Have a look at the comment I left.

**Comment by 1999** [comment]
@torifat I moved this PR to https://github.com/torifat/better-avj-errors/pull/4

**Pull request closed without merging**